examples/nextjs-next-image-loader/README.md

```diff
@@ -6,54 +6,67 @@ View Demo: <https://cloudinary-nextjs-next-image-loader.netlify.app>
 
 ## 🧰 Using the Cloudinary Loader with the Next.js Image Component
 
-> ⚠️ Note: This is not the  recommended approach. Use the CldImage component with [Next Cloudinary](https://next-cloudinary.spacejelly.dev) for first-class support of Cloudinary with the Next.js Image component.
+> ⚠️ Note: This is not the recommended approach. Use the CldImage component with [Next Cloudinary](https://next.cloudinary.dev) for first-class support of Cloudinary with the Next.js Image component.
 
 To use Cloudinary as a Loader with the Image component, you must specify a `loader` prop for each instance of an Image.
 
-```
-const normalizeSrc = (src) => src[0] === '/' ? src.slice(1) : src
+```js
+function normalizeSrc(src, width) {
+  // Optimization options cannot be taken as arguments in the loader function.
+  // To add optimization options like `q_auto`, `f_auto`, etc., add them to the src URL before the public ID.
+  return `w_${width}/` + src.replace(/^\//, "");
+}
 
-export function cloudinaryLoader({ src, width, quality }) {
-  const params = ['f_auto', 'c_limit', 'w_' + width, 'q_' + (quality || 'auto')];
-  return `https://res.cloudinary.com/${process.env.CLOUDINARY_CLOUD_NAME}/image/upload/${params.join(',')}/${normalizeSrc(src)}`;
+export function cloudinaryLoader({ src, width }) {
+  const cloudName = process.env.NEXT_PUBLIC_CLOUDINARY_CLOUD_NAME;
+  if (!cloudName) {
+    throw new Error(
+      "NEXT_PUBLIC_CLOUDINARY_CLOUD_NAME is required in the environment"
+    );
+  }
+  return `https://res.cloudinary.com/${cloudName}/image/upload/${normalizeSrc(src, width)}`;
 }
 
 <Image ... loader={cloudinaryLoader} />
 ```
 
+When using the new Next.js App Router, the `loader` prop must be specified in a client component. To solve this, create a wrapper client component like [components/Image.jsx](components/Image.jsx) to use in the App Router
+
 > Tip: As of Next.js 13.0.0, you can not globally specify a Loader.
```

Comment:
update: you can specify it globally as a loaderfile

https://nextjs.org/docs/app/api-reference/next-config-js/images

there's actually a separately maintained newer example that covers all of this that im realizing he added when we had an existing one: https://github.com/cloudinary-community/cloudinary-examples/tree/main/examples/nextjs-custom-image-loader

Reply:
Thanks for the heads up! I'll update the README to link to the newer example for global loader part